deeplabcut/utils/make_labeled_video.py

```diff
@@ -675,11 +674,9 @@ def create_labeled_video(
             shuffle,
             cfg,
             modelprefix,
-            engine=Engine.PYTORCH,
-        )
-        model_config_path = (
-            Path(config).parent / model_folder / "train" / Engine.PYTORCH.pose_cfg_name
         )
+        model_config_path = Path(config).parent / model_folder / "train"
```

Comment:
I think this still needs to be updated:

```suggestion
        model_config_path = Path(config).parent / model_folder / "train" / "pose_cfg.yaml"
```